docker-compose-tests.yml

```diff
@@ -10,7 +10,7 @@ services:
     volumes:
       - gp-finder-tests-node_modules:/code/node_modules
     environment:
-      NODE_ENV: test
+      NODE_ENV: ${NODE_ENV-test}
```

Comment:
@st3v3nhunt I see your point about the `test` bit. Is there a reason we use `development` for Travis though? The thing I'm trying to fish at is - can we make the 2 test scripts as similar as possible?